portable/NetworkInterface/board_family/ReadMe.txt

```diff
@@ -1 +1,5 @@
-Update NetworkInterface.c and include other files needed by FreeRTOS+TCP here.
\ No newline at end of file
+Update NetworkInterface.c and include other files needed by FreeRTOS+TCP here.
+
+Note: This particular NetworkInterface.c is Not! to be used as is. The purpose of this file is to provide a template for writing a network interface.
+Each network interface will have to provide concrete implemtations of the functions in this file.
```

Comment:
Is this correct here? Since this is a readme and there are no functions in this file.

Reply:
Updated readme.